packages/edit-site/src/components/block-editor/block-editor-provider/navigation-block-editor-provider.js

```diff
@@ -1,114 +0,0 @@
-/**
- * WordPress dependencies
- */
```

Comment:
@draganescu I'm trying to absorb the logic that we had here within the `EditorProvider` of `@wordpress/editor`. Can you help ensure I didn't break anything for the navigation pages in the site editor.